assets/config/0.5.4-coins.json

```diff
@@ -8560,6 +8560,58 @@
     "active": false,
     "currently_enabled": false
   },
+  "PRCY-BEP20": {
+    "coin": "PRCY-BEP20",
+    "name": "PRivaCY Coin",
+    "coinpaprika_id": "prcy-privacy-coin",
+    "coingecko_id": "privacy-coin",
+    "nomics_id": "prcy-privacy-coin",
```

Comment:
the nomics_id of PRCY, see https://nomics.com/assets/prcy-privacy-coin/widget (`data-base="PRCY"`)